### ISSUE TITLE
Issue with Connections (WWPN) in New-OVServerProfileTemplate

### ISSUE THREAD

**Issue by oolivera15** [HewlettPackard/POSH-HPOneView#601]
**Expected Behavior**
New-OVServerProfileTemplate should create the Connections with the WWPN information.

**Actual Behavior**

New-OVServerProfileTemplate create the server profile without the WWPN Connections information.
This information can be checked via the web interface where it shows "WWPN: pending assignment".

This situation causes an error when creating a server profile from the template via API:

```
"message": "Connection 1 does not have a WWPN specified. Fibre Channel connections must specify a WWPN for this server model.",
"nestedErrors": "",
"recommendedActions": "Specify the physical WWPN of the server adapter port to be used for this Fibre Channel connection."
```
**Steps to reproduce**
```

#Attributes for ServerProfileTemplate
$name                               = "server_template"
$shtName                            = "DL380p Gen8 1"
$sht                                = Get-OVServerHardwareType -Name $shtName

#Connections section
$cnConsistency                      = "Minimum"

#Attributes for connection
$connID                             = 1
$connName                           = "FC_A"
$connType                           = "FibreChannel"
$netName                            = "SAN_A"
$ThisNetwork                        = Get-OVNetwork -Type FibreChannel -Name $netName
$requestedMbps                      = "Auto"
$Conn1                              = New-OVServerProfileConnection -ConnectionID $connID -Name $connName -ConnectionType $connType -Network $ThisNetwork -arrayWWPN "XX:XX:XX:XX:XX:XX:XX:XX" -RequestedBW $requestedMbps 
New-OVServerProfileTemplate -Name $name -ServerHardwareType $sht -ManageConnections $True -ConnectionsConsistencyChecking $cnConsistency  -Connections $connections


#Attributes for ServerProfile
$name                               = "server_profile"
$server                             = Get-OVServer -Name "server_name"
$sptName                            = "server_template"
$spt                                = Get-OVServerProfileTemplate -Name "server_template"
New-OVServerProfile -Name $name -AssignmentType Server -Server $server -ServerProfileTemplate $spt
```

**Version Information**
Powershell:

```
> Get-Host | Select-Object Version

Version
-------
7.0.4
```

HPEOneView: 6.50.00-0452161

HPEOneView module:
```
> (Get-Module -Name HPEOneView.630).Version

Major  Minor  Build  Revision
-----  -----  -----  --------
6      30     2928   3678
```

Could you please help us to solve this issue?
Thank you!

**Comment by jullienl** [comment]
For DL servers, you are supposed to use the `FCConnectionAddresses` parameter to define the WWPN value that should be associated with a connection, for example :

```
$FCConnectionAddresses =  @{1 = $WWPN1; 2 = $WWPN2}

New-OVServerProfile -Name $SPname -AssignmentType Server -Server $server -ServerProfileTemplate $spt -FCConnectionAddresses $FCConnectionAddresses
```

**Comment by jullienl** [comment]
A fix is required to make the WWPN correctly assigned to the SP object, see #604

**Comment by oolivera15** [comment]
Hello.

The procedure is working now.
Will this new change be added to the HPEOneView module?

Thank you very much for your answer!

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 6.30.3188.1900](https://github.com/HewlettPackard/POSH-HPEOneView/releases/tag/v6.30.3188.1900) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPEOneView.630/6.30.3188.1900).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
